packages/@aws-cdk/aws-iotevents/lib/state.ts

```diff
@@ -55,11 +80,20 @@ export class State {
   }
 }
 
-function getEventJson(events: Event[]): CfnDetectorModel.EventProperty[] {
+function getEventJson(events: IEvent[]): CfnDetectorModel.EventProperty[] {
```

Comment:
Again, not a great name, and now that we have 3 functions and methods all starting with "get", this really starts to become hard to read. How about `toEventJson()`, to make it symmetric with `_toStateJson()`?